runhouse/rns/login.py

```diff
@@ -146,6 +146,15 @@ def login(
         # via configs.token = token
         configs.set("token", token)
 
+    autostop_mins = configs.defaults_cache.get("default_autostop")
+    if autostop_mins is None:
```

Comment:
should we download the config to local first before checking for and updating autostop defaults? for the case of logging in on a new device

Reply:
i guess if we move this check just a little further down we could do that? since if `download_config` is set to True thats what we'd do anyway